graylog2-web-interface/src/views/components/Select.jsx

```diff
@@ -115,24 +121,34 @@ const Select = ({ components, styles, ignoreCase = true, ignoreAccents = false,
     valueContainer,
   };
   const filterOption = createFilter({ ignoreCase, ignoreAccents });
+  const selectProps = {
```

Comment:
Extracting the props makes it harder to see how the effective props look like. Instead we could leave the structure as before (with inline props next to the component) and do something like: `const Component = allowOptionCreation ? CreatableSelect : ReactSelect;` and render `<Component ...` later.

Reply:
Thanks, this is a very good suggestion.